crates/bcr-wdc-keys/src/lib.rs

```diff
@@ -118,14 +128,59 @@ pub fn verify_with_keys(keyset: &cdk02::MintKeySet, proof: &cdk00::Proof) -> Res
     let keypair = keyset
         .keys
         .get(&proof.amount)
-        .ok_or(Error::NoKeyForAmount(proof.amount))?;
+        .ok_or(VerifyWithKeysError::NoKeyForAmount(proof.amount))?;
     cashu::dhke::verify_message(&keypair.secret_key, proof.c, proof.secret.as_bytes())?;
     Ok(())
 }
 
-pub fn into_secp256k1_msg(msg: &[u8]) -> bitcoin::secp256k1::Message {
-    let sha = Sha256::hash(msg);
-    bitcoin::secp256k1::Message::from_digest(*sha.as_ref())
+static SECPCTX: OnceLock<bitcoin::secp256k1::Secp256k1<bitcoin::secp256k1::All>> = OnceLock::new();
```

Comment:
Could you use this global context from the lib as well instead? https://docs.rs/secp256k1/latest/secp256k1/static.SECP256K1.html